docs/incremental-stage.md

```diff
@@ -0,0 +1,47 @@
+# Incremental staging
+
+**Stage 3** of `pvc-migrate` is responsible of running the rsync transfers for the PVCs discovered. An incremental **stage 3** might be desired to reduce the amount of data and time required during the final migration.
+
+## Overview
+
+There are three basic steps during an incremental staged migration :
+
+1. An incremental stage assumes you have finished the **initial** data sync by completing [Stage 1](../1_pvc_data_gen), [Stage2](../2_pvc_destination_gen) and [Stage3](../3_run_rsync) for the desired namespace(s)
+2. Perform incremental stage 3 rsync as many times as necessary
+3. Perform final stage 3 rsync with applications quiesced
```

Comment:
I thought we decided that before the final stage 3 rsync (quiesced re-run), the recommendation was to run all the stages again, wonder if we should clarify here?

Reply:
@alaypatel07 I think it would make sense to re-run all stages. Otherwise, if the Pod mounting the PVC on the source side changed, restarted or got rescheduled we will have lost correct references for a successful run.

Reply:
@alaypatel07 The final application quiesced re-run at the end will render their nodelists empty since the PVCs will not be attached, I can make a note regarding refreshing right before the application quiesce, so in that case they will catch any new changes and still have proper nodelists. The final migration has to be done with dummy deployments in order to have those PVCs available but that is a manual step at the moment.

Reply:
Corrected instructions